src/components/Item/MediaInfo.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <div class="text--secondary">
+    <span v-if="item.Type === 'Episode'">
+      {{ item.SeasonName }} &#8226; {{ item.IndexNumber }}
```

Comment:
I'd rather use the ``seasonEpisode`` or ``seasonEpisodeAbbrev`` translation keys for consistency.

Reply:
Of course, you are right, I had not noticed the presence of this key.

I will use "seasonEpisodeAbbrev" to avoid unnecessary word wraps in mobile!

I'll come back soon :)

Reply:
Ammend and force repush your commit please! 👍🏻

Reply:
I'm not entirely sure what you mean by " force repush your commit ".

I made a new commit, before reading your new comment and I wonder if you were asking me to force overwrite the previous commit (which in case, I didn't know was possible).

What did you mean? A simple new commit or an overwrite somehow I don't know about?

Reply:
@doc-code-hub Yes, exactly that.

You can overwrite your previous commit using (with the files previously staged): ``git commit --amend --no-edit``
If also want to change the commit message, just replace ``--no-edit`` with ``-m "new commit message"``

If you want to do this with other commits (not your previous one), you can run ``git log``, scroll down to the previous commit to the one where you want to make changes, copy the hash and do ``git rebase -i *hash*``

A text document with all the possible rebase options will appear for you. You can drop, reorder or squash commits this way (squash is exactly what I wanted, to keep a more clear history).

No worries, I'll do it for you.

Reply:
I see!

Helpful, I keep learning something from you from time to time.

I'll keep it in mind for the collections page I'm working on and use it instead of making a new commit for \#1680

Reply:
Making new commits per se it's not bad, the content itself of the commits is what you should be aware of :)

In this case, having two commits for a three line change is wasteful. But in your other PR, you can make commits for each stage you go by. Use a rule of thumb everything that's worth keeping for historical reasons. For instance:

1. Commit cleaning up unused vars from your page
2. Commit removing the component store
3. Commit migrating everything to the item page store.
4. Whatever else you think it's worth snapshotting.

This way we still keep your original implementation in the commit history, which might be useful in the future if we want to know why we ended up having a feature in X way instead of Y way.

So no worries to do commits, just made them when they're needed, and in this case it clearly wasn't :).